auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -498,14 +498,16 @@ def __init__(
         self.distributed = dist.is_initialized()
 
         if self.global_rank == 0:
-            print(f"Segmenter started  config_file:{config_file}, config_dict: {config_dict}")
+            print(f"Segmenter started  config_file: {config_file}, config_dict: {config_dict}")
 
         np.set_printoptions(formatter={"float": "{: 0.3f}".format}, suppress=True)
         logging.getLogger("torch.nn.parallel.distributed").setLevel(logging.WARNING)
 
 
         config = self.parse_input_config(config_file=config_file, override=config_dict)
         self.config = config
+        self.config_file = config_file if not isinstance(config_file, (list, tuple)) else config_file[0]
+        self.override = config_dict
```

Comment:
Would it be good to do a copy? `self.override = config_dict.copy()`

Reply:
It never changes , so it's ok as is.